assets/queries/k8s/limit_capabilities_of_the_psp/test/positive_expected_result.json

```diff
@@ -1,6 +1,6 @@
 [
 	{
-		"queryName": "Limit The Capabilities From The PSP",
+		"queryName": "Not Limited Capabilities For Pod Security Policy",
```

Comment:
Unlimited?

Reply:
Prefer the use of Not Limited in this case